style.css

```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&display=swap');
+
+body {
+  font-family: Inter, Helvetica, Geneva, sans-serif;
+}
+
+.top-header {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 12px;
+  margin-right: 40px;
+  align-items: center;
+}
+
+.welcome {
+  text-decoration: none;
+  color: #172b4d;
+  font-size: 20px;
+  font-weight: 600px;
+}
+
+.identical-to {
+  font-size: 40px;
+  width: 2px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-image: url('images/background.png');
+  background-size: cover;
+  background-size: 100% 100%;
+  padding: 58px;
+}
+
+.first-name {
+  font-weight: 700px;
+  font-size: 40px;
+}
+
+.software-dev {
+  color: #36b37e;
+}
+
+.social {
+  display: flex;
```

Comment:
### OPTIONAL
_I write it as an optional but this is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times) that is not mentioned in the last reviews._
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/)

Reply:
This is nice, it avaoid zero repeatifion in my CSS, thanks!